indico/htdocs/sass/base/_layout.scss

```diff
@@ -156,3 +156,58 @@ body > * {
 .flex-center {
     justify-content: center;
 }
+
+
+/* Flexbox layout */
+.flexrow {
```

Comment:
I still think this would be clearer:

> [18:37:52] <@ThiefMaster> flexrow -> .flex.row?
> [18:38:05] <@ThiefMaster> ie .flex to set display:flex and .col/.row to set the flow